### PR TITLE
Polish language

### PR DESCRIPTION
**Pull request by blackfrost79**
## Type
- Enhancement

## Pull request description
Add a Polish language version to Fork CMS

### CODE REVIEW

**Comment by blackfrost79** [comment]
I checked the indents before submitting the PR. Can this be caused by the editor (notepad++) I used?  I will go over them again tonight.

**Comment by jeroendesloovere** [comment]
We use spaces as indents.
I see that you used tabs.

**Comment by blackfrost79** [comment]
OK I'll go over them again once I have a bit of free time :)

**Comment by blackfrost79** [comment]
I think the indents have all been fixed.

File: src/Backend/Modules/Extensions/Installer/Data/locale.xml [hunk 1]
```diff
@@ -15,6 +15,7 @@
         <translation language="uk"><![CDATA[Шаблон для "%1$s" додано.]]></translation>
         <translation language="sv"><![CDATA[mall "%1$s" har lagts till.]]></translation>
         <translation language="el"><![CDATA[Το πρότυπο "%1$s" προστέθηκε.]]></translation>
+		<translation language="pl"><![CDATA[Szablon "%1$s" został dodany.]]></translation>
```
Comment (jeroendesloovere) [inline src/Backend/Modules/Extensions/Installer/Data/locale.xml]: Can you fix the indents in this entire file please?

File: src/Backend/Modules/Faq/Installer/Data/locale.xml [hunk 2]
```diff
@@ -13,6 +13,7 @@
         <translation language="sv"><![CDATA[lägg till fråga]]></translation>
         <translation language="el"><![CDATA[Νέα ερώτηση]]></translation>
         <translation language="zh"><![CDATA[添加问题]]></translation>
+		<translation language="pl"><![CDATA[dodaj pytanie]]></translation>
```
Comment (jeroendesloovere) [inline src/Backend/Modules/Faq/Installer/Data/locale.xml]: Can you fix the indents in this entire file please?

File: src/Backend/Modules/FormBuilder/Installer/Data/locale.xml [hunk 3]
```diff
@@ -16,6 +16,7 @@
         <translation language="lt"><![CDATA[nauja]]></translation>
         <translation language="sv"><![CDATA[lägg]]></translation>
         <translation language="el"><![CDATA[Προσθήκη]]></translation>
+		<translation language="pl"><![CDATA[dodaj]]></translation>
```
Comment (jeroendesloovere) [inline src/Backend/Modules/FormBuilder/Installer/Data/locale.xml]: Can you fix the indents in this entire file please?

File: src/Backend/Modules/Locale/Installer/Data/locale.xml [hunk 4]
```diff
@@ -15,6 +15,7 @@
         <translation language="uk"><![CDATA[Дії]]></translation>
         <translation language="lt"><![CDATA[veiksmai]]></translation>
         <translation language="el"><![CDATA[Ενέργειες]]></translation>
+		<translation language="pl"><![CDATA[Działania]]></translation>
```
Comment (jeroendesloovere) [inline src/Backend/Modules/Locale/Installer/Data/locale.xml]: Can you fix the indents in this entire file please?

File: src/Backend/Modules/Mailmotor/Installer/Data/locale.xml [hunk 5]
```diff
@@ -16,6 +16,7 @@
                 <translation language="sv"><![CDATA[mailmotor]]></translation>
                 <translation language="uk"><![CDATA[MailMotor]]></translation>
                 <translation language="zh"><![CDATA[邮件]]></translation>
+				<translation language="pl"><![CDATA[mailmotor]]></translation>
```
Comment (jeroendesloovere) [inline src/Backend/Modules/Mailmotor/Installer/Data/locale.xml]: Can you fix the indents in this entire file please?

File: src/Backend/Modules/MediaLibrary/Installer/Data/locale.xml [hunk 6]
```diff
@@ -6,657 +6,788 @@
         <translation language="en"><![CDATA[add media]]></translation>
         <translation language="nl"><![CDATA[bestanden toevoegen]]></translation>
         <translation language="ru"><![CDATA[добавить медиа]]></translation>
+		<translation language="pl"><![CDATA[dodaj media]]></translation>
```
Comment (jeroendesloovere) [inline src/Backend/Modules/MediaLibrary/Installer/Data/locale.xml]: Can you fix the indents in this entire file please?

File: src/Backend/Modules/Settings/Installer/Data/locale.xml [hunk 7]
```diff
@@ -15,6 +15,7 @@
         <translation language="uk"><![CDATA[ID Адміністратора]]></translation>
         <translation language="lt"><![CDATA[admino id]]></translation>
         <translation language="el"><![CDATA[ID του Διαχειριστή]]></translation>
+		<translation language="pl"><![CDATA[ID administratora]]></translation>
```
Comment (jeroendesloovere) [inline src/Backend/Modules/Settings/Installer/Data/locale.xml]: Can you fix the indents in this entire file please?

**Comment by jeroendesloovere** [comment]
Don't forgot to add "pl" to `src/Backend/Core/Installer/CoreInstaller.php`.
So people can select it when installing Fork CMS.

**Comment by jeroendesloovere** [comment]
Also do not forgot to create `src/Backend/Modules/Pages/Installer/Data/pl/` folder with translated files in it.

**Comment by blackfrost79** [comment]
As far as I can see everything is ready to go. But I've been wrong before ;) For the additional files, should I create another PR?

**Comment by jeroendesloovere** [comment]
Thanks for updating the locale.xml files.

1. Can you just add `pl` to
* src/Backend/Core/Installer/CoreInstaller.php
* src/Console/Locale/EnableLocaleCommand.php
* src/ForkCMS/Bundle/InstallerBundle/Form/Type/LanguagesType.php
2. Duplicate the pages files from "en" to "pl" (you don't need to translate them in this PR)

**Comment by blackfrost79** [comment]
I have them translated already. 

src/Backend/Core/Installer/CoreInstaller.php - already has a pl added to it. The data/pl/ was also already there I just translated the files inside. 

I just commited them to master.

**Comment by blackfrost79** [comment]
I removed the file.

**Pull request merged**